autogen/agentchat/contrib/retriever/__init__.py

```diff
@@ -0,0 +1,17 @@
+from typing import Optional
+from .chromadb import ChromaDB
+from .lancedb import LanceDB
```

Comment:
These two imports may need to move into the function so that any db can work without other dbs.